src/endpoints/journals/getJournalWorksByISSN.ts

```diff
@@ -0,0 +1,23 @@
+import axios, { AxiosResponse } from 'axios';
+import { endpoints } from '../../constants';
+import { RequestConfig } from '../../types/RequestConfig';
+import { createParams } from '../../utils/createParams';
+import { WorksMessage } from '../works/types';
+
+// Returns a list of works in the journal identified by {issn}.
+export const getJournalWorksByISSN = async (
+  id: string[],
+  config?: RequestConfig,
+) => {
+  try {
+    const { BASE_URL, JOURNALS, WORKS } = endpoints;
+    const params = createParams(config);
+    const url = `${BASE_URL + JOURNALS}${id}/${WORKS + params}`;
```

Comment:
You're combining + operator with backticks, just use backticks for everything IE

`
${BASE_URL}${JOURNALS}${id}/${WORKS}${params}
`